keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -206,24 +225,22 @@ CNP resources, such that no two CNP instances can be created with the same `prio
 (TODO: Is such admission control viable?)
```

Comment:
This is what we need to discuss in today's meeting. If we do not reach a conclusion, at least we need to move the TODOs into a separate section as Dan suggested

Reply:
+1 I'm making an out of band list of things I would like to discuss I'll make sure to add this one